.github/CHANGELOG.md

```diff
@@ -114,6 +114,10 @@ which uses the old Numba code. When setting to a higher value, the new Julia cod
 [(#309)](https://github.com/XanaduAI/MrMustard/pull/309)
 * Fixing a bug where `scipy.linalg.sqrtm` returns an unsupported type.
 [(#337)](https://github.com/XanaduAI/MrMustard/pull/337)
+* Fixing a bug in `_transform_gaussian` in transformation.py that modifies the input state's cov and means.
```

Comment:
This is not in the right place. Should be in the top section of the file (0.7.1)

Reply:
Got it! I'll move it to there